ARrehab/ARrehab/Player.swift

```diff
@@ -25,20 +28,21 @@ class Player : TileCollider, HasModel, HasAnchoring{
     
     override func onCollisionBegan(tile: Tile) {
         self.onTile = tile
-        super.onCollisionBegan(tile: tile)
+        //self.onTile.changeMaterials(materials: [SimpleMaterial(color: .blue, isMetallic: false)])
+        //super.onCollisionBegan(tile: tile)
     }
     
     override func onCollisionEnded(tile: Tile) {
-        if self.onTile == tile {
-            self.onTile = nil
```

Comment:
Is `self.onTile` not used anymore?

Reply:
It is, but one of the issues that Woojin pointed out (finicky board outline, would appear / disappear almost randomly) was being caused by our augmentation of self.onTile in the onCollisionEnded method. I found it worked much better if I just updated self.onTile when the collision began (and do nothing when collision ends)